src/components/navigation/card/components/CardContent.tsx

```diff
@@ -4,12 +4,14 @@ import classNames from 'classnames';
 import CardContext from '../CardContext';
 
 const CardContent: React.FC<HTMLProps<HTMLDivElement>> = ({ className, ...rest }) => {
-  const { feature } = useContext(CardContext);
+  const { feature, primary, secondary } = useContext(CardContext);
```

Comment:
possibly change this into a type parameter instead of 3 bools which can be 'feature' | 'primary' | 'secondary | undefined

Reply:
Done